pyanaconda/modules/payloads/source/hmc/hmc_module.py

```diff
@@ -0,0 +1,67 @@
+#
```

Comment:
Do you have a reason to call this `/hmc/hmc_module` instead of `/hmc/hmc` as other sources are?

Reply:
I have started to use that in the Storage module, because it makes the file structure easier to understand, but I can change it back.

Reply:
It would be great to stay consistent in this. If we want to go the `_module` way let's create an issue to rename everything or stay as it is now ideally.

Reply:
Ok, fixed.